backstop/tests/modal.html

```diff
@@ -40,13 +40,21 @@ <h1>Modal</h1>
 
     <a
       id="open-modal"
-      onclick="openModal()"
+      onclick="toggleModal('myModal')"
```

Comment:
Really strange modal id 😀 Can you rename it to `default-modal` maybe?

Reply:
Done